src/core/Footer/Footer.tsx

```diff
@@ -49,43 +60,70 @@ const footerTranslations: TitleTranslations = {
  * Footer element with all needed legal and info links.
  * Be sure to place it manually at the bottom of your page.
  * You can use position 'absolute' with relative body or set the height of the content and place footer at the end.
- * @example
+ * @example <caption>Appending custom element after default elements</caption>
  * <Footer customElements={[{title: 'Bentley', url: 'https://www.bentley.com/'}]} />
+ * @example <caption>Returning only custom elements</caption>
+ * <Footer customElements={() => newFooterElements)} />
+ * @example <caption>Filtering out a specific element</caption>
+ * <Footer customElements={(defaultElements: FooterElement[]) => defaultElements.filter(({ key }) => key !== 'privacy' )} />
+ * @example <caption>Changing a url</caption>
+ * <Footer customElements={(defaultElements: FooterElement[]) => defaultElements.map(element => ({ ...element, url: element.key === 'privacy' ? customPrivacyUrl : element.url }))} />
  */
 export const Footer = (props: FooterProps) => {
   const { customElements, translatedTitles, className, ...rest } = props;
 
   useTheme();
 
-  const today = new Date();
   const titles = { ...footerTranslations, ...translatedTitles };
-  const defaultElements = [
+  const defaultElements: FooterElement[] = [
+    {
+      key: 'copyright',
+      title: titles.copyright,
+    },
     {
+      key: 'termsOfService',
       title: titles.termsOfService,
       url:
         'https://connect-agreementportal.bentley.com/AgreementApp/Home/Eula/view/readonly/BentleyConnect',
     },
-    { title: titles.privacy, url: 'https://www.bentley.com/en/privacy-policy' },
     {
+      key: 'privacy',
+      title: titles.privacy,
+      url: 'https://www.bentley.com/en/privacy-policy',
+    },
+    {
+      key: 'termsOfUse',
       title: titles.termsOfUse,
       url:
         'https://www.bentley.com/en/terms-of-use-and-select-online-agreement',
     },
-    { title: titles.cookies, url: 'https://www.bentley.com/en/cookie-policy' },
-    { title: titles.legalNotices, url: 'https://connect.bentley.com/Legal' },
+    {
+      key: 'cookies',
+      title: titles.cookies,
+      url: 'https://www.bentley.com/en/cookie-policy',
+    },
+    {
+      key: 'legalNotices',
+      title: titles.legalNotices,
+      url: 'https://connect.bentley.com/Legal',
+    },
   ];
-  const elements = customElements
-    ? [...defaultElements, ...customElements]
-    : defaultElements;
+
+  let elements: FooterElement[] = defaultElements;
+  if (customElements) {
+    elements =
+      typeof customElements === 'function'
+        ? customElements(defaultElements)
+        : [...defaultElements, ...customElements];
+  }
 
   return (
     <footer className={cx('iui-legal-footer', className)} {...rest}>
       <ul>
-        <li>© {today.getFullYear()} Bentley Systems, Incorporated</li>
         {elements.map((element, index) => {
           return (
             <li key={`${element.title}-${index}`}>
```

Comment:
the key from elements can be used now
```suggestion
            <li key={key || `${element.title}-${index}`}>
```

Reply:
These are going to be different values than before. "termsOfService" instead of "Terms of service - 1" Is that ok?

Reply:
I did it anyway.